### ISSUE TITLE
TypeError: Cannot read property 'target_temperature' of null

### ISSUE THREAD

**Issue by mskg** [mskg/homey-heating#96]
https://sentry.io/organizations/mskg/issues/907336332/?project=1396900&referrer=github_plugin

```
TypeError: Cannot read property 'target_temperature' of null
  File "./src/app/services/device-manager/DeviceManagerService.ts", line 221, in h.<anonymous>
    device.watchedCapabilities.targetTemperature = await device.makeCapabilityInstance<number>(CapabilityType.TargetTemperature, (evt) => {
  ?, in Generator.next
  ?, in new Promise
  File "./src/app/services/device-manager/DeviceManagerService.ts", line 213, in apply
    private async attachWatchers(device: AuditedDevice) {
  File "./src/app/services/log/trycatchlog.ts", line 41, in attachWatchers
    const result = func.apply(this, args);
...
(5 additional frame(s) were not displayed)

TypeError: Cannot read property 'target_temperature' of null
```

**Comment by mskg** [comment]
Seems to be related to https://github.com/athombv/homey-web-api-issues/issues/3

**Comment by mskg** [comment]
Should be resolved by #95, see comments in athombv/homey-web-api-issues#3.
